doc/source/doc-style/docstrings.rst

```diff
@@ -102,7 +102,7 @@ Parameters
 ----------
 Functions and class methods may have parameters in their signatures. All these
```

Comment:
I still don't know how to distinguish between methods and functions.

Reply:
Indeed, the two are very similar! To be precise with the language:

- Function is the general term, it includes both *methods* and *free functions*:
- *methods* are functions which are "attached" to a class
- *free functions* in contrast do not belong to a class

The sentence above also mentions "class methods" -- which is again a subset of "methods"; they are attached to a class, but can be called from the class directly, as opposed to "instance methods", which need an _instance_ of a class.

In more relaxed / sloppy speech, "function" can refer to "free function" if it's clear from context.

So again, as a tree

- Functions
  - Free functions
  - Methods
    - Class methods
    - Instance methods
    - Static methods

In general, the [Python glossary](https://docs.python.org/3/glossary.html) is a good place to look up such terms. It's a bit annoying, but each programming language can have slightly different ways in which things are named.

Long story short: For the sentence above, I think it would be sufficient to say

``Functions can have parameters...``

To be pedantic, classes _also_ have signatures, but that is the same as the signature of their constructor (a special method).